docs/components/DocumentationPage.tsx

```diff
@@ -117,7 +117,12 @@ export default function DocumentationPage(props: Props) {
       isMobileMenuVisible={isMobileMenuVisible}
       onContentScroll={handleContentScroll}
       sidebarScrollPosition={sidebarScrollPosition}>
-      <Head title={props.title} description={props.description}>
+      <Head
+        title={props.title}
+        description={props.description}
+        canonicalUrl={
+          version !== 'latest' && version !== 'unversioned' ? getCanonicalUrl(pathname) : ''
```

Comment:
```suggestion
          version !== 'latest' && version !== 'unversioned' ? getCanonicalUrl(pathname) : undefined
```

We can use `undefined` here instead.